VogtPayroll5/PayrollConsoleReader.cs

```diff
@@ -24,5 +24,31 @@ public Employee GetEmployeeInfo()
 
             return emp;
         }
+
+        public bool ReturnTrueOrFalse()
+        {
+            string temp;
+            Console.WriteLine("Would you like to add another employee? y/n");
+            temp = Console.ReadLine().ToLower();
+
+            do
+            {
+                if (temp == "y")
+                {
+                    return true;
+                }
+                else if (temp == "n")
+                {
+                    return false;
+                }
+                else
+                {
```

Comment:
Something that took me a little bit to pick up on, but in the above if statements you're always returning a value -- the function **ends**.

Therefore, you don't need the else here.

You can simplify with this:
```c#
     do
            {
                if (temp == "y")
                {
                    return true;
                }
                else if (temp == "n")
                {
                    return false;
                }
                Console.WriteLine("Please enter 'y' or 'n'!");
                temp = Console.ReadLine().ToLower();
            } while (temp != "y" || temp == "n");
```